spark/src/main/java/org/apache/iceberg/spark/source/SparkAppenderFactory.java

```diff
@@ -205,6 +205,7 @@ private StructType lazyPosDeleteSparkType() {
         case PARQUET:
           return Parquet.writeDeletes(file.encryptingOutputFile())
               .createWriterFunc(msgType -> SparkParquetWriters.buildWriter(lazyEqDeleteSparkType(), msgType))
+              .setAll(properties)
```

Comment:
Thanks for the contribution, @coolderli !  I also think the newPosDeleteWriter need the properties setting ...

Reply:
@coolderli, could you add this update?

Reply:
@zhong-yj xxx